forgottenserver_800/doc/CONFIG_HELP.txt
[ CONFIG HELP
	Project Name
		The Forgotten Server

	Version
		0.3

	Codenamed
		Crying Damson

	License
		GNU GPLv3

	Forum
		http://otland.net/
]

[ ABOUT
	List with all, possible to use in config.lua file, config values.
	It contains info, possible and default value.
]

[ LIST
	*ACCOUNT MANAGER
		accountManager
			Defines if account manager should be enabled.
			Disable it if you're using website and don't need such feature (it's a part of AAC).
			Default: yes

		namelockManager
			Defines if namelock manager should be enabled.
			If you enable this feature, player who has been namelocked
				will be able to login on account manager and change their names.
			Default: no

		newPlayerChooseVoc
			Ignore if your account manager is disabled.
			Default: no

		newPlayerSpawnPosX
			Ignore if your account manager is disabled.
			Default: 100

		newPlayerSpawnPosY
			Ignore if your account manager is disabled.
			Default: 100

		newPlayerSpawnPosZ
			Ignore if your account manager is disabled.
			Default: 7

		newPlayerTownId
			Ignore if your account manager is disabled.
			Default: 1

		newPlayerLevel
			Ignore if your account manager is disabled.
			Default: 1

		newPlayerMagicLevel
			Ignore if your account manager is disabled.
			Default: 0

		generateAccountNumber
			Default: yes

	*UNJUSTIFIED KILLS
		redSkullLength
			Default: 30 * 24 * 60 * 60

		blackSkullLength
			Default: 45 * 24 * 60 * 60

		dailyFragsToRedSkull
			Default: 3

		weeklyFragsToRedSkull
			Default: 5

		monthlyFragsToRedSkull
			Default: 10

		dailyFragsToBlackSkull
			Default: dailyFragsToRedSkull

		weeklyFragsToBlackSkull
			Default: weeklyFragsToRedSkull

		monthlyFragsToBlackSkull
			Default: monthlyFragsToRedSkull

		dailyFragsToBanishment
			Default: dailyFragsToRedSkull

		weeklyFragsToBanishment
			Default: weeklyFragsToRedSkull

		monthlyFragsToBanishment
			Default: monthlyFragsToRedSkull

		blackSkulledDeathHealth
			Default: 40

		blackSkulledDeathMana
			Default: 0

		useBlackSkull
			New feature from 8.5. If you'll use it, player won't be banished but instead they receive black skull which block some actions.
			Default: yes

		advancedFragList
			Default: no

		useFragHandler
			Enable or disable hardcoded frag handler.
			Default: yes

	*BANISHMENTS
		notationsToBan
			Amount of notations to receive by player to be banned automatically.
			Default: 3

		warningsToFinalBan
			Default: 4

		warningsToDeletion
			Default: 5

		banLength
			Ban length in seconds.
			Default: 7 * 24 * 60 * 60 (7 days)

		killsBanLength
			Ban length for unjustified player killing.
			Works only if useBlackSkull option is disabled.
			Default: 7 * 24 * 60 * 60 (7 days)

		finalBanLength
			Default: 30 * 24 * 60 * 60 (30 days)

		ipBanishmentLength
			Default: 1 * 24 * 60 * 60 (1 day)

		broadcastBanishments
			Specify if every banishment should be broadcasted (red message).
			Default: yes

		maxViolationCommentSize
			Default: 200

		violationNameReportActionType
			1 = just a report, 2 = name lock, 3 = player banishment
			Default: 2

		autoBanishUnknownBytes
			Specify if player should be banned automatically in case his client send unknown byte to server.
			It happens mostly when player use unofficial software to play.
			Default: no

	*BATTLE
		worldType
			Values: pvp, no-pvp, pvp-enforced
			Default: pvp

		protectionLevel
			To this level players are protected.
			It means they cannot be attacked by players neither anyone can attack them.
			Default: 1 (Means, that there is no level protection!)

		pvpTileIgnoreLevelAndVocationProtection
			Specify if tiles with pvp-zone flag are ignoring protection checks.
			If it's enabled and players are fighting on pvpTiles:
				Some of them have lower level than protectionLevel - they're able to fight.
				Some of them have protected vocation (E.q. none vocation) - they're able to fight.
			Default: yes

		pzLocked
			Time needed to wait after attacking a player.
			In miliseconds.
			Default: 60 * 1000 (60 seconds)

		huntingDuration
			In miliseconds.
			Default: 60 * 1000

		criticalHitChance
			Chance in percent to inflict a critical hit.
			Default: 7

		criticalHitMultiplier
			If critical hit occurs, its strength will be multiplied by this value.
			Default: 1

		displayCriticalHitNotify
			Display text effect when critical hit occurs? ("CRITICAL!")
			Default: no

		removeWeaponAmmunition
			Should player ammunition be removed? Ex. bolts - should remove their charges when used?
			Default: yes

		removeWeaponCharges
			Default: yes

		removeRuneCharges
			Should rune charges be removed when using? No, means that they will have infinite charges.
			Default: yes

		whiteSkullTime
			In miliseconds.
			Default: 15 * 60 * 1000 (15 minutes)

		noDamageToSameLookfeet
			If enabled, players with same outfit feet color will be unable to attack themselfes.
			It works only for melee attacks.
			Default: no

		showHealingDamage
			Display effect with health points gained when creature was healed?
			Default: no

		showHealingDamageForMonsters
			Should healing effect was also displayed for monsters?
			Default: yes

		fieldOwnershipDuration
			How long a creature is an owner of a field item created by him.
			In miliseconds.
			Default: 5 * 1000

		stopAttackingAtExit
			If enabled, when player exits the game without logout, he'll stop attacking already attacked creatures.
			Good feature when you have trainers and want to disable a possibilty to train after closing Tibia by exit.
			Default: no

		oldConditionAccuracy
			Default: no

		loginProtectionPeriod
			The famous Tibia anti-magebomb system
			Amount of time during which player is unable to attack other players after his login.
			In miliseconds.
			Default: 10 * 1000 (10 seconds)

		deathLostPercent
			deathLostPercent set to nil enables manual mode.
			Default: 10

		stairhopDelay
			Feature introduced in 8.41 global tibia update.
			Specify time, which player need to wait after changing floor/teleport etc, before he is able to attack monster.
			In miliseconds.
			Default: 2 * 1000

		pushCreatureDelay
			How fast creatures are pushed.
			In miliseconds.
			Default: 2 * 1000

		deathContainerId
			Default container player gets after death, when he have empty container slot.
			Default: 1987

		gainExperienceColor
			Color used when creature gain experience.
			Values: From 0 to 255.
			Value will be randomized if you use -1
			Default: 215

		addManaSpentInPVPZone
			Should player be able to train magic level in PVP zones?
			Protection against mana training in PVP Arena, where mana is re-newed when player dies.
			Default: yes

		squareColor
			Color of square visible when attacking creature.
			Value will be randomized if you use -1
			Default: 0

		allowFightback
			System introduced in 8.5, if you enable it then players who defend themselves from the attacks of other players do not get a protection zone block.
			Default: yes

	*CONNECTION
		worldId
			Change only if you want to use multiworld feature.
			Default: 0

		ip
			ip address of a server.
			Default: 127.0.0.1

		bindOnlyConfiguredIpAddress
			Default: no

		loginPort
			Port where login server will listen.
			Login server means server which responds with character list.
			Default: 7171

		gamePort
			Game service requires to run on seperated port!
			DO NOT run this service with ANY other on SAME PORT.
			Any merging with other ports will end with services not working properly,
			so please keep in mind the default configuration that is easy and works.
			Default: 7172

		adminPort
			Port where admin protocol will listen.
			Default: 7171

		statusPort
			Port where status protocol will listen.
			Default: 7171

		loginTries
			How much tries specified IP address has to login.
			E.g. after 10 wrong passwords, IP will be blocked until next server restart.
			Default: 10

		retryTimeout
			In miliseconds.
			Default: 5 * 1000

		loginTimeout
			In miliseconds.
			Default: 60 * 1000

		maxPlayers
			Online players limit.
			Default: 1000

		motd
			Message Of The Day.
			This message in sent to client before displaying characters list.
			NOTE: Its displayed only once but when you change this message it will be displayed more than once.
			Default: Welcome to the Forgotten Server!

		displayOnOrOffAtCharlist
			Should character status be specified when displaying characters list? (Online/Offline)
			If enabled, serverName won't be displayed on list.
			Default: no

		onePlayerOnlinePerAccount
			If enabled, only one player per account will be able to be online.
			Default: yes

		allowClones
			Allow multiple client logins on one character.
			Mostly used on pvp-enfo servers where you have a few characters
				and all of them are able to login on one character, e.g 'Master Sorcerer'
			Default: no

		serverName
			Default: Forgotten

		loginMessage
			Message displayed in console (in white color) when characters appear in the game.
			Default: Welcome to the Forgotten Server!

		statusTimeout
			Minimal interval for specified IP address to request server status again.
			Use lower value, if you use website which is requesting status frequently.
			In miliseconds.
			Default: 5 * 60 * 1000 (5 hours)

		replaceKickOnLogin
			If player is online and other player tries to connect,
				should player in game be kicked and new client be able to connect and replace him?
			Default: yes

		forceSlowConnectionsToDisconnect
			Clients with slow connection will be disconnected automatically.
			Default: no

		loginOnlyWithLoginServer
			Enable it only if you're using multi world system. This is just a game server and you won't enable to get characters list.
			Default: no

		premiumPlayerSkipWaitList
			Default: no

	*DATABASE
		NOTE: sqlFile is used only by sqlite database, and sqlKeepAlive by mysql database.
		To disable sqlKeepAlive such as mysqlReadTimeout use 0 value.

		sqlType
			Database storage engine.
			Values: mysql, odbc, sqlite, pgsql
			Default: sqlite

		sqlHost
			Default: localhost

		sqlPort
			Default: 3306

		sqlUser
			Default: root

		sqlPass
			Default: ""

		sqlDatabase
			Default: theforgottenserver

		sqlFile
			Used only by sqlite database.
			Default: forgottenserver.s3db

		sqlKeepAlive
			Used only by mysql database.
			Use 0 to disable.
			Default: 0

		mysqlReadTimeout
			Use 0 to disable.
			Default: 10

		mysqlWriteTimeout
			Use 0 to disable.
			Default: 10

		encryptionType
			Defines how passwords should be stored in database.
			*WARNING: Using plain passwords is not recommented!*
			Values: plain, md5, sha1
			Default: plain

	*DEATHLIST
		deathListEnabled
			Should players' deaths be storaged in database?
			Default: yes

		deathListRequiredTime
			Default: 1 * 60 * 1000

		deathAssistCount
			Default: 20

		maxDeathRecords
			Player deathlist limit.
			Ignored if deathList is disabled.
			Default: 5

	*GUILDS
		ingameGuildManagement
			Default: "yes"

		levelToFormGuild
			Level required to form a guild.
			Default: 8

		premiumDaysToFormGuild
			Premium days required to form a guild.
			Default: 0

		guildNameMinLength
			Minimum length of a guild name.
			Default: 4

		guildNameMaxLength = 20
			Maximum length of a guild name.
			Default: 20

	*HIGHSCORES
		highscoreDisplayPlayers
			How much players will be displayed in highscores.
			Default: 15

		updateHighscoresAfterMinutes
			How often highscores list will be updated.
			In minutes.
			Default: 60

	*HOUSES
		buyableAndSellableHouses
			Ingame house system enabled?
			Default: yes

		houseNeedPremium
			Is premium required to buy a house?
			NOTE: Houses are cleaned everytime server start,
				so if this option is enabled and player don't have premium - house will be removed. (If houseRentPeriod != never)
			Default: yes

		bedsRequirePremium
			Do players need premium account to use beds?
			Default: yes

		levelToBuyHouse
			Level needed to buy a house.
			Default: 1

		housesPerAccount
			How many houses can be bought per one account.
			Default: 0 (means no limit!)

		houseRentAsPrice
			Default: no

		housePriceAsRent
			Default: no

		housePriceEachSquare
			House price
			Default: 1000

		houseRentPeriod
			Values: yearly, monthly, weekly, daily, never
			Default: never

		houseCleanOld
			Defines if houses should be auto cleaned when player is inactive (Not logging in specified amount of time)
			In seconds, 0 to disable.
			Default: 0

		guildHalls
			Use guild halls system, allowing to buy guild hall only bu guild leader. All members will be auto invited.
				You can mark house as guild hall in map editor.
			Default: no

	*ITEM USAGE
		timeBetweenActions
			Default: 200

		timeBetweenExActions
			Time player need to wait before using next item.
			Default: 1000

		checkCorpseOwner
			Defines if corpse should be checked for owner before opening.
			So for first x seconds, only creature killer is able to open it.
			Default: yes

		hotkeyAimbotEnabled
			Should system with attacking via hotkeys and battle window be enabled? (Cipsoft antibot features)
			Default: yes

		maximumDoorLevel
			Default: 500

	*MAP
		mapName
			Map name which server will load.
			NOTE: if map file is named 'somemap.otbm' then just paste here 'somemap'.
			Default: forgotten

		mapAuthor
			Map author. Used by status protocol (ots lists)
			Default: Unknown

		randomizeTiles
			Specify if all items should be randomized while creating (like grass).
			You can configure this in data/XML/randomization.xml file.
			Default: yes

		useHouseDataStorage
			New house data storage system. Performs much faster map saving. (up to 60x faster!)
			If enabled - items will be saved binary in `house_data` table, otherwise `tiles` and `tile_items` will be used.
			Default: no

		storeTrash
			Define if trash items should be stored in memory.
			It costs more memory but will perform a lot faster cleaning.
			Otherwise, if you disable this option, cleaning will be performed by going through all the tiles
				and looking for trash. (which of course consumes more CPU resources)
			Default: yes

		cleanProtectedZones
			Should protection zone tiles be cleaned at map cleaning?
			Default: yes

		mailboxDisabledTowns
			Town IDs where player can't send parcel/letters.
			Example "1,5,9"
			Default: -1 (Means, no towns disabled)

	*STARTUP
		defaultPriority
			Works only on Windows
			Values: high, higher, realtime
			Default: high

		niceLevel
			Default: 5

		coresUsed
			Specifies a cores of CPU that are used to run TFS
			Put a cores' numbers, counting from 0 and separate them with comma
			(fe. coresUsed = "0, 1, 2, 3" means you have a quad core
				and want TFS to use all of its cores)
			Default: -1

		optimizeDatabaseAtStartup
			Should be database optimized at server startup?
			Recommended - yes!
			Default: yes

		removePremiumOnInit
			Should accounts premium time be updated at server start-up?
			Default: yes

		confirmOutdatedVersion
			Decides if server should display a box (GUI) or a message (console)
			while starting up when current version is not up-to-date
			and wait for user interaction (yes/no) until continuing start-up.
			Default: yes

	*MUTED BUFFER
		maxMessageBuffer
			Default: 4

		bufferMutedOnSpellFailure
			Default: no

	*MISCELLANEOUS
		dataDirectory
			Directory, from where all data used be server is loaded.
			Default: data/

		allowChangeOutfit
			Should player's be able to change their outfits?
			Default: yes

		allowChangeColors
			Should player's be able to change their outfit colors?
			Default: yes

		allowChangeAddons
			Should player's be able to change their outfit addons?
			Default: yes

		disableOutfitsForPrivilegedPlayers
			Are privileged players able to change their outfits?
			E.g gamemasters have default outfits and shouldn't be able to change them.
			Default: no

		bankSystem
			Use bank system?
			E.g if enabled, rent for a house will be taken from player's bank account.
			Default: yes

		saveGlobalStorage
		
			Global storages are stored in `global_storage` table.
			Default: yes

		displaySkillLevelOnAdvance
			Default: no

		spellNameInsteadOfWords
			E.g if you use 'exura', it will show 'Light Healing'.
			Default: no

		emoteSpells
			If enabled, spells wont be visible in 'Default' channel, but as orange effect.
			Anti spam feature.
			Default: no

		promptExceptionTracerErrorBox
			Works only on Windows with precompiled support feature,
				"exception tracer" (__EXCEPTION_TRACER__ flag)
			Default: "yes"

		storePlayerDirection
			Should players look direction be saved and restored while logging?
			In global tibia it's disabled.
			Default: no

		monsterLootMessage
			Defines how green message with loot in the middle of screen should be displayed.
				0 - disable this message
				1 - enable only in party
				2 - enable only for player
				3 - both, if party available show there, otherwise send to player (global Tibia)
					+ you can enable/disable it individually for each monster by using <flag lootmessage="yes/no"/>
			Default: 3

		monsterLootMessageType
			Type/id(color) of loot message.
			You can use values from 18 to 27.
			Default: 25 (green)

	*GHOST MODE
		ghostModeInvisibleEffect
			Should invisible gamemaster be visible to other gamemasters with invisible effect? (Like, utana vid spell)
			If you disable this, you'll see all ghost gamemasters in normal outfits.
			Default: no

		ghostModeSpellEffects
			Display effects while casting spells in ghost mode?
			If 'no', then no one will see that ghost GM casted a spell.
			Default: yes

	*LIMITS
		idleWarningTime
			Default: 14 * 60 * 1000

		idleKickTime
			Default:15 * 60 * 1000

		expireReportsAfterReads
			Default: 1

		playerQueryDeepness
			Default: 2

		maxItemsPerPZTile
			Maximum items which can be placed on 1 protection zone tile.
			Default: 0 (means no limit!)

		maxItemsPerHouseTile
			Maximum items which can be placed on 1 house tile.
			Default: 0 (means no limit!)

	*PREMIUM-RELATED
		freePremium
			Should all players have free premium accounts?
			NOTE: It's just enabling it in game without any changes in a database.
			Default: no

		premiumForPromotion
			Do players need premium to buy promotion?
			Default: yes

	*BLESSINGS
		blessingsOnlyPremium
			Should only premium players be able to use blessings?
			Default: yes

		blessingReductionBase
			Default: 30

		blessingReductionDecreament
			Default: 5

		eachBlessReduction
			NOTE: blessingReduction* regards items/containers loss.
			eachBlessReduction determine by how many percentage points does each bless decrease an experience/magic/skills loss.
			Default: 8

	*RATES
		experienceStages
			If you enable this option, you can edit experience stages in file data/XML/stages.xml
			Default: no

		rateExperience
			How much multiplied will be experience gained from monsters.
			This option is ignored, if experienceStages are enabled.
			Default: 5.0

		rateExperienceFromPlayers
			System, used in global Tibia, on pvp-enforced servers.
			Specify how much experience should player gain by killing other players.
			Experience is only received if _killer_ level is lower than _killed_ level.
			0 to disable.
			Default: 0

		rateSkill
			Default: 3.0

		rateMagic
			Default: 3.0

		rateLoot
			How much faster loot will be dropped from monsters.
			Default: 2.0

		rateSpawn
			Default: 1

		rateMonsterHealth
			Monster health multiplier.
			For example, if monster have 1000 health points by default,
				and you set this value to 2.0 - then monster will have 2000 health points.
			Default: 1.0

		rateMonsterMana
			Monster mana cost multiplier.
			Means how much more mana points player need to summon/convince a monster.
			Default: 1.0

		rateMonsterAttack
			Monster attack strength multiplier.
			Default: 1.0

		rateMonsterDefense
			Monster defense multiplier.
			Default: 1.0

	*EXPERIENCE FROM PLAYERS
		minLevelThresholdForKilledPlayer
			Default: 0.9

		maxLevelThresholdForKilledPlayer
			Default: 1.1

	*STAMINA
		NOTE: Stamina is stored in miliseconds so you must multiply seconds by 1000.
		rateStaminaHits multiplies every hit a creature does which is later
		multiplied by player's attack speed.
		rateStaminaGain is divider of every logged out second, eg:
		60000 / 3 = 20000 milliseconds, what gives 20 stamina seconds for 1 minute being logged off.
		rateStaminaThresholdGain is divider for the premium stamina.
		staminaRatingLimit* is in minutes.

		rateStaminaLoss
			Default: 1

		rateStaminaGain
			Default: 3

		rateStaminaThresholdGain
			Default: 12

		staminaRatingLimitTop
			In minutes.
			Default: 41 * 60

		staminaRatingLimitBottom
			In minutes.
			Default: 14 * 60

		rateStaminaAboveNormal
			Default: 1.5

		rateStaminaUnderNormal
			Default: 0.5

		staminaThresholdOnlyPremium
			Default: yes

	*PARTY
		experienceShareRadiusX
			Max range (X) between players, to be experience sharing could be active.
			Default: 30

		experienceShareRadiusY
			Max range (Y) between players, to be experience sharing could be active.
			Default: 30

		experienceShareRadiusZ
			Max range (Z) between players, to be experience sharing could be active.
			Default: 1

		experienceShareLevelDifference
			NOTE: float number, highestLevel * value
			Default: 2 / 3

		extraPartyExperienceLimit
			Default: 20

		extraPartyExperiencePercent
			Default: 5

		experienceShareActivity
			Default: 2 * 60 * 1000

	*GLOBAL SAVE
		NOTE: globalSaveHour means like 03:00, not that it will save every 3 hours,
		if you want such a system please check out data/globalevents/globalevents.xml.

		globalSaveEnabled
			Should daily saving system aka global Tibia be enabled?
			Default: no

		globalSaveHour
			Hour when server will be closed.
			Default: 8

		shutdownAtGlobalSave
			Defines if server should be closed at global save.
			Default: yes

		cleanMapAtGlobalSave
			Defines if map should be cleaned at global save.
			NOTE: Its ignored when shutdownAtGlobalSave is enabled.
			Default: no

	*SPAWNS
		deSpawnRange
			Default: 2

		deSpawnRadius
			Default: 50

	*SUMMONS
		maxPlayerSummons
			Amount of summons players are able to have.
			Default: 2

		teleportAllSummons
			Should summons be teleported to monsters?
			Default: no

		teleportPlayerSummons
			Should summons be teleported to thir owners? (E.g. if owner changes a floor level)
			Default: no

	*STATUS
		ownerName
			Default: ""

		ownerEmail
			Default: @otland.net

		url
			Default: http://otland.net/

		location
			Default: Europe

		displayGamemastersWithOnlineCommand
			Should online gamemasters be displayed when using !online command?
			Default: no

	*LOGS
		displayPlayersLogging
			Should message in console be displayed everytime player logs in/out?
			Default: yes

		prefixChannelLogs
			Defines channel logs file prefix.
			Usable if you have multiworld, and want to have channels from all worlds in different files.
			Default: ""

		adminLogsEnabled
			Create logs from remote control tool.
			Default: no

		NOTE: A kind of logging listed below does not work in GUI version.
		For such, please compile the software with __GUI_LOGS__ flag.

		runFile
			It will save process pid to this file if specified.
			Default: ""

		outLogName
			Default: ""

		errorLogName
			Default: ""

		truncateLogsOnStartup
			Default: no
]


792/htdocs/config/config.php
<?PHP
# Account Maker Config
$config['site']['serverPath'] = "C:/Users/shynz/Documents/GitHub/otserv/792/";
$config['site']['useServerConfigCache'] = false;
$towns_list = array(1 => 'Enigma City');

$config['site']['outfit_images_url'] = 'http://outfit-images.ots.me/outfit.php';
$config['site']['item_images_url'] = 'http://item-images.ots.me/960/';
$config['site']['item_images_extension'] = '.gif';
$config['site']['flag_images_url'] = 'http://flag-images.ots.me/';
$config['site']['flag_images_extension'] = '.png';

# Create Account Options
$config['site']['one_email'] = false;
$config['site']['create_account_verify_mail'] = false;
$config['site']['verify_code'] = true;
$config['site']['email_days_to_change'] = 3;
$config['site']['newaccount_premdays'] = 0;
$config['site']['send_register_email'] = false;

/* if you eanle this option you will add
extra values to new character and it starts at level 8
*/
$config['site']['add_extra_info'] = true;

# Create Character Options
$config['site']['newchar_vocations'] = array(1 => 'Sorcerer Sample', 2 => 'Druid Sample', 3 => 'Paladin Sample', 4 => 'Knight Sample');
$config['site']['newchar_towns'] = array(1);
$config['site']['max_players_per_account'] = 7;


# Emails Config
$config['site']['send_emails'] = false;
$config['site']['mail_address'] = "xxxx@gmx.com";
$config['site']['smtp_enabled'] = true;
$config['site']['smtp_host'] = "mail.gmx.com";
$config['site']['smtp_port'] = 25;
$config['site']['smtp_auth'] = false;
$config['site']['smtp_user'] = "xxx@gmx.com";
$config['site']['smtp_pass'] = "xxxx";

# PAGE: whoisonline.php
$config['site']['private-servlist.com_server_id'] = 0;
/*
Server id on 'private-servlist.com' to show Players Online Chart (whoisonline.php page), set 0 to disable Chart feature.
To use this feature you must register on 'private-servlist.com' and add your server.
Format: number, 0 [disable] or higher
*/

# PAGE: characters.php
$config['site']['quests'] = array();
$config['site']['show_skills_info'] = true;
$config['site']['show_vip_storage'] = 0;

# PAGE: accountmanagement.php
$config['site']['send_mail_when_change_password'] = true;
$config['site']['send_mail_when_generate_reckey'] = true;
$config['site']['generate_new_reckey'] = false;
$config['site']['generate_new_reckey_price'] = 500;

# PAGE: guilds.php
$config['site']['guild_need_level'] = 15;
$config['site']['guild_need_pacc'] = false;
$config['site']['guild_image_size_kb'] = 50;
$config['site']['guild_description_chars_limit'] = 2000;
$config['site']['guild_description_lines_limit'] = 6;
$config['site']['guild_motd_chars_limit'] = 250;

# PAGE: adminpanel.php
$config['site']['access_admin_panel'] = 3;

# PAGE: latestnews.php
$config['site']['news_limit'] = 6;

# PAGE: killstatistics.php
$config['site']['last_deaths_limit'] = 40;

# PAGE: team.php
$config['site']['groups_support'] = array(2, 3, 4, 5, 6);

# PAGE: highscores.php
$config['site']['groups_hidden'] = array(4, 5, 6);
$config['site']['accounts_hidden'] = array(1);

# PAGE: shopsystem.php
$config['site']['shop_system'] = false;

# PAGE: lostaccount.php
$config['site']['email_lai_sec_interval'] = 180;

# Layout Config
$config['site']['layout'] = 'tibiacom';
$config['site']['vdarkborder'] = '#505050';
$config['site']['darkborder'] = '#D4C0A1';
$config['site']['lightborder'] = '#F1E0C6';
$config['site']['download_page'] = false;
$config['site']['serverinfo_page'] = true;